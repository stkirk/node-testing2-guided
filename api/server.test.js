const server = require('./server')
const request = require('supertest')
const db = require('../data/dbConfig')

beforeAll(async () => {
  await db.migrate.rollback()
  await db.migrate.latest()
})

beforeEach(async () => {
  await db.seed.run()
})

afterAll(async () => {
  await db.destroy()
})
