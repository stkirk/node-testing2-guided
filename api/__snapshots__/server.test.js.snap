// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[POST] /hobbits responds with the newly created hobbit 1`] = `
Object {
  "id": 5,
  "lady": "gaga",
  "name": "bilbo",
}
`;
